Huntarr

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Huntarr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/huntarr.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/9.1.5/80cf92a64bef6889 (running kernel: 6.17.9-1-pve)

### 📝 Provide a clear and concise description of the issue.

Receive the below error when trying to install. 



### 🔄 Steps to reproduce the issue.

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/huntarr.sh)"

### ❌ Paste the full error output (if available).

    __                __                 
   / /_  __  ______  / /_____ ___________
  / __ \/ / / / __ \/ __/ __ `/ ___/ ___/
 / / / / /_/ / / / / /_/ /_/ / /  / /    
/_/ /_/\__,_/_/ /_/\__/\__,_/_/  /_/     
                                         
  ⚙️  Using Default Settings on node pve
  💡  PVE Version 9.1.5 (Kernel: 6.17.9-1-pve)
  🆔  Container ID: 109
  🖥️  Operating System: debian (13)
  📦  Container Type: Unprivileged
  💾  Disk Size: 4 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 1024 MiB
  🚀  Creating a huntarr LXC using the above default settings
  
build_container
  ✔️   Storage local (Free: 69.4GB  Used: 19.8GB) [Template]
  ✔️   Storage local-lvm (Free: 802.5GB  Used: 13.8GB) [Container]
  ✔️   Storage 'local-lvm' (lvmthin) validated
  ✔️   Template storage 'local' validated
  ✔️   Template search completed
  ✔️   Template debian-13-standard_13.1-2_amd64.tar.zst [local]
  ✔️   LXC Container 109 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 192.168.1.225 
  ✔️   IPv4 Internet Connected
  ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
  ✔️   Updated Container OS
  ✔️   Python 3.12 installed
  ✔️   Setup uv 0.10.4
  ✔️   Deployed: huntarr (9.3.7)
  ✖️   in line 21: exit code 1 (General error / Operation not permitted)
 → uv pip install --python /opt/huntarr/.venv/bin/python -r /opt/huntarr/requirements.txt

  --- Last 10 lines of log ---
              See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
              ********************************************************************************

      !!
        self._finalize_license_expression()
      error: command 'c++' failed: No such file or directory

      hint: This usually indicates a problem with the package or the build environment.
[2026-02-20 14:38:14] [ERROR] in line 21: exit code 1 (General error / Operation not permitted)
[2026-02-20 14:38:14] uv pip install --python /opt/huntarr/.venv/bin/python -r /opt/huntarr/requirements.txt
  -----------------------------------

  ✖️   Installation failed in container 109 (exit code: 1)

[1]+  Stopped                 bash -v -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/main/ct/huntarr.sh)"

### 🖼️ Additional context (optional).

_No response_

--- a/ct/huntarr.sh
+++ b/ct/huntarr.sh
@@ -35,7 +35,8 @@ function update_script() {
     msg_info "Stopping Service"
     systemctl stop huntarr
     msg_ok "Stopped Service"
-
+    
+    ensure_dependencies git
     fetch_and_deploy_gh_release "huntarr" "plexguide/Huntarr.io" "tarball"
 
     msg_info "Updating Huntarr"

--- a/ct/huntarr.sh
+++ b/ct/huntarr.sh
@@ -36,7 +36,7 @@ function update_script() {
     systemctl stop huntarr
     msg_ok "Stopped Service"
     
-    ensure_dependencies git
+    ensure_dependencies build-essential
     fetch_and_deploy_gh_release "huntarr" "plexguide/Huntarr.io" "tarball"
 
     msg_info "Updating Huntarr"

--- a/install/huntarr-install.sh
+++ b/install/huntarr-install.sh
@@ -14,7 +14,7 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y build-essential
+$STD apt install -y build-essential
 msg_ok "Installed Dependencies"
 
 PYTHON_VERSION="3.12" setup_uv

--- a/install/huntarr-install.sh
+++ b/install/huntarr-install.sh
@@ -13,6 +13,10 @@ setting_up_container
 network_check
 update_os
 
+msg_info "Installing Dependencies"
+$STD apt-get install -y build-essential
+msg_ok "Installed Dependencies"
+
 PYTHON_VERSION="3.12" setup_uv
 fetch_and_deploy_gh_release "huntarr" "plexguide/Huntarr.io" "tarball"
 

